### ISSUE TITLE
[BUG] prest crashes if user set a cache directory that does not exists in filesystem

### ISSUE THREAD

**Issue by moinologics** [prest/prest#858]
steps to reproduce

- pull the latest docker image
- enable cache & set some cache directory (which not exists in filesystem, for example `/some/non-existing/path`) in prest.toml
- start the server
- hit the http endpoint (first time it will show error)

you will see like this in response

```
PANIC: runtime error: invalid memory address or nil pointer dereference
goroutine 33 [running]:
github.com/urfave/negroni/v3.(*Recovery).ServeHTTP.func1()
	/home/runner/go/pkg/mod/github.com/urfave/negroni/v3@v3.0.0/recovery.go:168 +0x140
panic({0x51c020, 0x9c3c10})
	/opt/hostedtoolcache/go/1.20.8/x64/src/runtime/panic.go:884 +0x1f4
github.com/tidwall/buntdb.(*Tx).lock(0x400022a440?)
	/home/runner/go/pkg/mod/github.com/tidwall/buntdb@v1.3.0/buntdb.go:1140 +0x24
```

when you hit url second time, it will not crash but will create cache files in /app instead of creating cache path setted by user in prest.toml file

**Comment by arxdsilva** [comment]
@moinologics I've raised a PR for a potential solution, but could you also provide the full stack trace of the panic error?

Thanks

**Comment by arxdsilva** [comment]
released on `v1.4.2`
